util/gen_hunt_data.py

```diff
@@ -4,12 +4,20 @@
 import io
 import coinach
 
+# Path to FFXIV installation (None == Default location)
+_FFXIV_GAME_PATH = None
```

Comment:
Could you make these command line args or looking at environment variables? Or, something that folks don't have to edit files directly in order to adjust?

Reply:
yeah just wanted a quick fix for the moment but will copy the style of the other tools to keep them consistent :)